Sources/SmithyIdentity/CRTAdapters/AWSCredentialIdentityUtilityInitializers.swift

```diff
@@ -37,6 +37,7 @@ public extension CRTAWSCredentialIdentity {
             accessKey: awsCredentialIdentity.accessKey,
             secret: awsCredentialIdentity.secret,
             sessionToken: awsCredentialIdentity.sessionToken,
+            accountId: awsCredentialIdentity.accountID,
```

Comment:
Account ID is now passed into the CRT credentials when they are created from SDK credentials.